test/test_main.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ * @brief     Test file
+ * @author    Jeroen van Hattem
+ * @license   MIT
+ */
+#define CATCH_CONFIG_MAIN // This tells Catch to provide a main() - only do this in one cpp file
+#include "../src/coordinate3d.hpp"
+#include "../src/robot_arm.hpp"
```

Comment:
using `../src` is disadvised. Most up-to-date project files allow including as `#include "coordinate3d.hpp"` directly